app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckExtractionService.scala

```diff
@@ -49,9 +50,11 @@ class HecTaxCheckExtractionServiceImpl @Inject() (
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
-  val licenceType: (String, String)           = ("licence-type", "LICENCE_TYPE")
-  val licenceTimeTrading: (String, String)    = ("licence-time-trading", "LICENCE_TIME_TRADING")
-  val licenceValidityPeriod: (String, String) = ("licence-validity-period", "LICENCE_VALIDITY_PERIOD")
+  val licenceType: FileDetails[LicenceType]                     = FileDetails[LicenceType]("licence-type", "LICENCE_TYPE")
```

Comment:
sorry, I named the files in the ticket incorrectly - they should be `HEC_LICENCE_TYPE`, `HEC_LICENCE_TIME_TRADING` and `HEC_LICENCE_VALIDITY_PERIOD` like it says in the spec